src/coverlet.core/coverlet.core.csproj

```diff
@@ -9,7 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="0.10.1" />
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-    <PackageReference Include="System.Reflection.Metadata" Version="1.6.0" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.5.0" />
```

Comment:
@vagisha-nidhi can add a comment above like `// Do not change System.Reflection.Metadata version we need to support VSTest DataCollectors` and maybe link some resource where we can check min version if exists

Reply:
@vagisha-nidhi I mean if there is a link where we can check minimal version of System.Reflection.Metadata to support DataCollectors

Reply:
There isn't any such link as per my knowledge.

Reply:
ok no problem

Reply:
Here is the link which shows object model dependency on System.Reflection.Metadata [https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj#L37](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj#L37)
Will this work?

Reply:
Could be, but reports 1.3.0 and you amend to 1.5.0 can you explain why?

Reply:
The assembly version has only been changes in 1.6.0 that is, 1.4.3.0
Both 1.3.0 and 1.5.0 have the same assembly version(1.4.2.0) and works as expected. So thought of bumping just 1 version down.

Reply:
Silly question you're right(verified)

Reply:
I'll edit the link to point to the object model csproj for reference? Then this would be good to go, right?

Reply:
yes, apply my comments please and to me is ok, we need to wait @tonerdo for merge.